webpack.config.js

```diff
@@ -139,4 +139,36 @@ module.exports = [
             'system': '{}',
             'file': '{}'
         }
+    }, 
+
+    // plugin target.
+    {
+        debug: true,
+        target: 'web',
+        node: {
+            fs: false
+        },
+        entry: {
+            'dfa-interpreter': './plugins/dfa-interpreter'
+        },
+        output: {
+            path: './build/plugins',
+            filename: '[name].js',
+            sourceMapFilename: '[name].js.map',
+            chunkFilename: '[id].chunk.js',
+            library: 'module'
+        },
+        resolve: {
+            extensions: ['.ts']
+        },
+
+        module: {
+            loaders: [
+                {
+                    test: /dfa-interpreter.ts$/,
```

Comment:
This can probably be all .ts files.
The way webpack works is it looks at your entry, then all of its requires, and so on... All those files are then sent through the loaders.
So specifying all .ts files here won't ruin anything. (probably.)